.github/workflows/go.yml

```diff
@@ -0,0 +1,54 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
```

Comment:
As far as I see that project uses go modules but not `go dep`... Could you elaborate on why then it's here?

Reply:
It is probably because it is in the default go workflow action.  It shouldn't harm anything.  Though.

Reply:
Agree that it shouldn't harm anything because it even checks if Gopkg.toml exist which does not in our case. So, it's a bit redundant and I would vote for removing if there is no reasons to keep it.

Reply:
I am fine with removing it.

Reply:
I'm fine removing it as well, kept it as it was default suggestion for a github action for go.
Removed it in 9fb6014